dist/leaflet.distortableimage.js

```diff
@@ -1629,28 +1635,99 @@ L.ExportAction = L.EditAction.extend({
     options.toolbarIcon = {
       svg: true,
       html: 'get_app',
-      tooltip: tooltip,
+      tooltip: tooltip
```

Comment:
we use dangling commas for objects
```suggestion
      tooltip: tooltip,
```

Reply:
:+1: